client/src/components/Header/Header.scss

```diff
@@ -0,0 +1,14 @@
+@import 'styles/variables';
+
+.Header {
+  display: flex;
+  flex-wrap: wrap;
+
+  .Header--Icon {
```

Comment:
I'm not a fan of the `.block--element-modifier` naming system.

Maybe something like `block__element--modifier` ?

Reply:
I don't really mind so let's do this 👍